plutus-core/plutus-core/test/Spec.hs

```diff
@@ -44,10 +45,11 @@ import Test.Tasty.Hedgehog
 main :: IO ()
 main = do
     plcFiles <- findByExtension [".plc"] "plutus-core/test/data"
```

Comment:
Ugh, I still really dislike the way these tests are organized...

Reply:
We could put these in their own `Spec` file, since we also have a property-based debruijn test in `Names/Spec.hs`